examples/all-clusters-app/esp32/main/main.cpp

```diff
@@ -109,7 +109,10 @@ constexpr EndpointId kNetworkCommissioningEndpointSecondary = 0xFFFE;
 static void InitServer(intptr_t context)
 {
     // Init ZCL Data Model and CHIP App Server
-    chip::Server::GetInstance().Init(&sCallbacks);
+    static chip::CommonCaseDeviceServerInitParams initParams;
+    (void) initParams.InitializeStaticResourcesBeforeServerInit();
```

Comment:
What is the rationale for burying the return value if `InitializeStaticResourcesBeforeServerInit` can actually fail?